### PR TITLE
install: document hypervisor tools

### PR DESCRIPTION
**Pull request by gsanchietti**
NethServer/nethsecurity#816

Replaces: #121

### CODE REVIEW

File: install.rst
Comment (filippocarletti) [inline install.rst:129]:
```suggestion
QEMU guest agent is not part of the NethSecurity image but can be installed from the command line.
```

File: install.rst
Comment (filippocarletti) [inline install.rst:167]:
```suggestion
VMware open-vm-tools are not part of the NethSecurity image but can be installed from the command line.
```

**Review by filippocarletti — CHANGES_REQUESTED** [review]

**Review by filippocarletti — APPROVED** [review]

**Comment by gsanchietti** [comment]
Wait for release before merge.

**Pull request merged**
